src/main/webapp/SearchResults.js

```diff
@@ -29,162 +59,67 @@ function loading() {
-          ' with views of 19th-century landmarks.',
-      id: 102,
-    },
-    {
-      country: 'Malaysia',
-      name: 'Langkawi',
-      description: 'Langkawi, the Jewel of Kedah, is a district and an ' +
-          'archipelago of 99 islands in the Andaman Sea some 30 km off the' +
-          ' mainland coast of northwestern Malaysia.',
-      id: 103,
-    },
-  ];
-  displayResults(results);
-}
-
-/**
- * loads Brazil info to page.
- */
-function brazilResults() {
-  const headingContainer = document.getElementById('Heading');
-  headingContainer.innerText = 'Brazil';
-
-  const results = [
-    {
-      country: 'Brazil',
-      name: 'Salvador',
-      description: 'Salvador, the capital of Brazil’s' +
-          ' northeastern state of Bahia, is known for its Portuguese colonial' +
-          ' architecture, Afro-Brazilian culture and a tropical coastline.',
-      id: 201,
-    },
-    {
-      country: 'Brazil',
-      name: 'Rio de Janeiro',
-      description: 'Rio de Janeiro is a huge seaside' +
-          ' city in Brazil, famed for its Copacabana and Ipanema beaches,' +
-          ' 38m Christ the Redeemer statue atop Mount Corcovado and for' +
-          ' Sugarloaf Mountain, a granite peak with cable cars to its summit.',
-      id: 202,
-    },
-    {
-      country: 'Brazil',
-      name: 'Langkawi',
-      description: 'Salvador, the capital of Brazil’s' +
-          ' northeastern state of Bahia, is known for its Portuguese colonial' +
-          ' architecture, Afro-Brazilian culture and a tropical coastline.',
-      id: 203,
-    },
-  ];
-  displayResults(results);
-}
-
-/**
- * Loads South Korea info to page.
- */
-function southKoreaResults() {
-  const headingContainer = document.getElementById('Heading');
-  headingContainer.innerText = 'SOUTH KOREA';
-
-  const results = [
-    {
-      country: 'South Korea',
-      name: 'Seoul',
-      description: 'Seoul, the capital of South Korea,' +
-          ' is a huge metropolis where modern skyscrapers, high-tech subways' +
-          ' and pop culture meet Buddhist temples, palaces and street markets.',
-      id: 201,
-    },
-    {
-      country: 'South Korea',
-      name: 'Busan',
-      description: 'Busan, a large port city in South' +
-          ' Korea, is known for its beaches, mountains and temples.',
-      id: 202,
-    },
-    {
-      country: 'South Korea',
-      name: 'Jeju-do',
-      description: 'Jeju province encompasses the ' +
-          'South Korean island of Jeju in the Korea Strait. Its known for' +
-          'its beach resorts and volcanic landscape of craters and cavelike' +
-          ' lava tubes.',
-      id: 203,
-    },
-  ];
-  displayResults(results);
-}
-
 /**
  * @param {results} results
  */
-function displayResults(results) {
-  // Create a flex-container div if necessary or grab last flex-container
+function displayCityResults(results, countries) {
```

Comment:
With a list of SearchResult JSON objects, the second parameter can be removed 
```suggestion
function displayResults(results) {
```